docs/developer/meta/ci/validation.md

```diff
@@ -0,0 +1,128 @@
+# Validation
+
+-----
+
+Various validations are [ran](https://github.com/DataDog/integrations-core/blob/master/.github/workflows/validate.yml) to check for correctness. There is a [reusable workflow](https://github.com/DataDog/integrations-core/blob/master/.github/workflows/run-validations.yml) that repositories may call with input parameters defining which validations to use, with each input parameter corresponding to a subcommand under the `ddev validate` command group.
+
+## Agent requirements
+
+```
+ddev validate agent-reqs
+```
+
+This validates that each integration version is in sync with the [`requirements-agent-release.txt`](https://github.com/DataDog/integrations-core/blob/master/requirements-agent-release.txt) file. It is uncommon for this to fail because the release process is automated.
+
+## CI configuration
+
+```
+ddev validate ci
+```
+
+This validates that all CI entries for integrations are valid. This includes checking if the integration has the correct [Codecov config](https://github.com/DataDog/integrations-core/blob/master/.codecov.yml), and has a valid [CI entry](testing.md#target-enumeration) if it is testable.
+
+!!! tip
+    Run `ddev validate ci --sync` to resolve most errors.
+
+## Codeowners
+
+```
+ddev validate codeowners
+```
+
+This validates that every integration has a [codeowner entry](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners). If this validation fails, add an entry in the [codewners file](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS) corresponding to any newly added integration.
+
+!!! note
+    This validation is only enabled for [integrations-extras](https://github.com/DataDog/integrations-extras).
+
+## Default configuration files
```

Comment:
Do we also want to mention `ddev validate models`? I see that some other commands are missing (such as package or recommended monitors) but I'm not such we want to list them all here

Reply:
I'm going to open a separate issue to update this page